### PR TITLE
Toggle dropdown arrow

### PR DESCRIPTION
**Pull request by jonest**
This resolves issue #53

dropdown

### CODE REVIEW

**Comment by jonest** [comment]
@andrew-frueh Good find! I see it on Firefox on Windows as well.

**Comment by jonest** [comment]
After looking into it, it's specifically related to this bit of CSS:

```css
&.closed::after {
    content: "\f078";
}
&.open::after {
    content: "\f077";
}
```

If I remove that, it won't toggle, but it won't close immediately. Something about changing this content causes it to close. I tried some various other things, but I'm stumped. I'm not sure what to do to get it to work in Firefox :(

**Comment by jonest** [comment]
@andrew-frueh It looks like it has something to do with the `hidden` attribute. If I remove that it toggles just fine. How important is it to keep that in there for the placeholder?

image

**Comment by corytak** [comment]
@jonest FYI I'm still seeing the error in Firefox where the dropdown closes immediately after opening.

**Comment by andrew-frueh** [comment]
Thanks for persisting on this one @jonest - the good news is, that on my laptop it looks like your change did fix the issue with the dropdown not staying open.  Unfortunately, what I'm still seeing on Chrome and Safari is that after you close the dropdown or make a selection, the arrow stays upside down until I click away from the select box (remove focus from it).  I think it has something to do with what fires the (focusout) command.  Like Chrome keeps focus on the select item even after you make a selection.

I would say mess around with other events to catch that, but I was just talking with @corytak and he was seeing some other strange behavior on his computer with the select box still not staying open.  I'll let Cory elaborate on what he was seeing.

Bottom line, I hate to see you spinning your wheels on this one when there is other stuff you could be picking off that are higher priority (like say #78 breadcrumbs, etc).  It seems like the click handlers kind of open a can of worms across browsers.  So maybe just better to put this one on the shelf and come back to it later?

**Comment by jonest** [comment]
> So maybe just better to put this one on the shelf and come back to it later?

It's no big deal! I haven't really been putting too much effort into it :P

> @jonest FYI I'm still seeing the error in Firefox where the dropdown closes immediately after opening.

@corytak Are you on Windows? I was on a Mac last night looking at this.


I'll take a peek at the issues at some point.

**Comment by corytak** [comment]
@jonest yes I'm on windows.
